frontend/packages/component-library/src/actionBlock/ActionBlock.tsx

```diff
@@ -0,0 +1,235 @@
+import classNames from 'classnames';
+import {HTMLAttributes} from 'react';
+
+import {LinkButton} from '@/button';
+import {
+  Heading3,
+  BodyThreeText,
+  OverlineTwoText,
+  StrongText,
+} from '@/typography';
+
+import moduleStyles from './actionBlock.module.scss';
+
+export interface ActionBlockProps extends HTMLAttributes<HTMLDivElement> {
+  /** Action Block title */
+  title: string;
+  /** Action Block description */
+  description: string;
+  /** Action Block image */
+  image?: string;
+  /** Action Block overline */
+  overline?: string;
+  /** Detail label */
+  detailLabel?: string;
+  /** Detail text */
+  detailDescription?: string;
```

Comment:
```suggestion
  /** Action Block Details */
  details?: {
    /** Detail label */
    label: string;
    /** Detail text */
    description: string;
  }
```

I think this should be more easier to read and use